### PR TITLE
Prevent panic on fuzzer provided string

### PR DESCRIPTION
**Pull request by zeripath**
The fuzzer has found that providing a <body> tag with an attribute to
PostProcess causes a panic. This PR removes any rendered html or body
tags from the output.

Replaces #14402

Signed-off-by: Andrew Thornton <art27@cantab.net>

### CODE REVIEW

File: modules/markup/html.go [hunk 1]
```diff
@@ -317,8 +317,8 @@ func RenderEmoji(
 	return ctx.postProcess(rawHTML)
 }
 
-var byteBodyTag = []byte("<body>")
-var byteBodyTagClosing = []byte("</body>")
+const byteBodyTag = "<body>"
```
Comment (zeripath) [inline modules/markup/html.go]:
```suggestion
```

File: modules/markup/html.go [hunk 2]
```diff
@@ -317,8 +317,8 @@ func RenderEmoji(
...
+const byteBodyTagClosing = "</body>"
```
Comment (zeripath) [inline modules/markup/html.go]:
```suggestion
```

**Comment by zeripath** [comment]
@lunny - the body wraparound is necessary.

**Review by 6543 — APPROVED** [review]

**Review by lunny — APPROVED** [review]

**Comment by 6543** [comment]
Backport: #14409

**Pull request merged**